.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,66 @@
+name: Integration tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      nodeCount:
+        description: 'Number of nodes in a cluster'
+        required: false
+        default: '3'
+
+jobs:
+  test:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-1
+
+      - name: Set up Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.12.25
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install jq
```

Comment:
Please remember about `haveged` if you want to generate keys on the fly

Reply:
Yes, I remember about this tool. But I left out generating keys for later.